### ISSUE TITLE
Share your comment on social media

### ISSUE THREAD

**Issue by dsernst** [unitedvote/united.vote#5]
After you vote, provide easy links to directly share your comment on social media.

**Comment by dsernst** [comment]
- [ ] Blocked by needing dedicated pages for individual comments first.

**Comment by dsernst** [comment]
For public votes:

`https://united.vote/legislation/:bill_id/votes/david`

And if they don't have a public vote yet, could say something like:

> ### David Ernst hasn't publicly voted on this issue.
>
> [Click here]() to ask them to leave a public vote.

With the link to a precomposed tweet: 

> Can you weigh in on HR 5323 — The Important Bill Name of 2018, @dsernst?

**Comment by dsernst** [comment]
For the share icons:

> ### Share ${own_page ? 'your'  : 'this'} comment
>
> facebook icon  |   twitter icon   |  email envelope

With pre-written text:

I'm voting [in favor / against] [bill number] [bill name]. See why: [link to comment].

**Comment by dsernst** [comment]
Kicking ass and taking names

**Comment by alexmingoia** [comment]
The comments have been combined into a single component, share icons have been added, and your comment is now highlighted on the vote page.

I'd like to hold off on the URL changes, as it needs more thought discussion. One issue is that anonymous comments would need to have separate logic to be linked to, and if that person later made it public or changed their username the URL will change or require redirects. It seems much simpler to use the comment id. Another issue is that in the future when we add comment replies, a user will possibly have multiple comments for a bill, so it's not clear how the URLs will work with that. Is it really necessary to have usernames in the URL?

Regarding the redirection to the comment page after voting, now that the comment is highlighted on the vote page I don't think that's necessary?
